### PR TITLE
Updated Abode docs to point to jaraco.abode library

### PR DESCRIPTION
**Pull request by jaraco**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/abode.markdown [hunk 1]
```diff
@@ -81,8 +81,8 @@ All {% term events %} have the fields:
 | `date` | The date of the event in the format `MM/DD/YYYY`. |
 | `time` | The time of the event in the format `HH:MM AM`. |
 
-There is a unique list of known event_codes that can be found
-[here](https://github.com/MisterWil/abodepy/files/1262019/timeline_events.txt).
+There is a unique list of known event_codes are defined in
```
Comment (c0ffeeca7) [inline source/_integrations/abode.markdown:84]: Why is the link to abodepy removed?

Comment (jaraco) [reply source/_integrations/abode.markdown:84]: I had checked the URL and thought it was 404. Turns out it wasn't 404, but when I first browsed to the URL, I included the trailing `)`, which _was_ 404, then when I corrected it and removed the trailing `)`, Firefox quietly downloaded the file and left me on the 404 page (leading me to believe the URL itself was 404).

So I sought to find a suitable replacement in the `jaraco.abode` project, which supersedes the abodepy project. Now that I see the contents of timeline_events.txt, I see there is a bit of extra information in that text file, information which is lost outside of the previous project's GitHub repo. I'll restore that content in the project itself.

Comment (jaraco) [reply source/_integrations/abode.markdown:84]: Aha. I see now that the "groups" is codified as well, so I'll update the docs to link to that.

Comment (jaraco) [reply source/_integrations/abode.markdown:84]: Done in cf6e9d6f6480ce51fe41efb4aabd74e12e71c21e.

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @jaraco :+1:

File: source/_integrations/abode.markdown [hunk 2]
```diff
@@ -92,7 +94,7 @@ Available {% term services %}: `change_setting`, `capture_image`, `trigger_autom
 
 Change settings on your Abode system.
 For a full list of settings and valid values, consult the
-[AbodePy settings section](https://github.com/MisterWil/abodepy/blob/master/README.rst#settings).
+[jaraco.abode settings section](https://github.com/jaraco/jaraco.abode/blob/main/README.rst#settings).
```
Comment (c0ffeeca7) [inline source/_integrations/abode.markdown]:
```suggestion
[`jaraco.abode` settings section](https://github.com/jaraco/jaraco.abode/blob/main/README.rst#settings).
```

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**